### ISSUE TITLE
text in "history" pane illegible with darker GUI skins

### ISSUE THREAD

**Issue by IndrajeetPatil** [Open-Systems-Pharmacology/PK-Sim#1872]
For example, I am working with skin "Visual Studio Code 2013". 

The problem occurs because of text with white color on a white background.

![Capture](https://user-images.githubusercontent.com/11330453/135452303-e1a81357-f731-4f09-81c4-a960c01c5b04.PNG)

**Comment by msevestre** [comment]
what about prameter views? (open an Individual. select an entry in the gree view

**Comment by IndrajeetPatil** [comment]
Haven't reached parameter views yet, but I also see the same issue for "Create Individual" modules

![individual](https://user-images.githubusercontent.com/11330453/135466521-e18b8586-1811-4239-866f-4f7e0a683d38.PNG)

**Comment by msevestre** [comment]
Yes this is the default disabled color

Can you update the color in your user settings to see if this solves the issue?

**Comment by IndrajeetPatil** [comment]
Indeed, that does solve the issue! 

Not sure if we should make the color settings adapt to the chosen GUI skin color?

![image](https://user-images.githubusercontent.com/11330453/135467831-2a10f4ee-fdbc-41db-b7be-becba63dcef8.png)

**Comment by msevestre** [comment]
@IndrajeetPatil Yes we probably should. Especially if the color did not change from DEFAULT colors.

**Comment by IndrajeetPatil** [comment]
This issue is being closed as part of a routine cleanup of issues I opened more than four years ago that remain unresolved. If you consider it still relevant, please feel free to reopen it.
